gammapy/irf/background.py

```diff
@@ -11,7 +11,7 @@
 from .core import IRF
 from .io import gadf_is_pointlike
 
-__all__ = ["Background3D", "Background2D"]
+__all__ = ["Background3D", "Background2D", "BackgroundIRF"]
```

Comment:
Why do you want to expose `BackgroundIRF`? Because of documentation?

Reply:
I think presence in `__all__` is required for it to show up in the API documentation.

I think the best way here would be to expand the docstring of the IRF baseclasses to explain to developers what is necessary to implement a new implementation.